MODULE.bazel

```diff
@@ -0,0 +1,62 @@
+module(
+    name = "pachyderm",
+    repo_name = "com_github_pachyderm_pachyderm",
```

Comment:
Just trying to improve my understanding of Bazel, why specify `repo_name`? My understanding¹ is that this is basically backwards compatibility feature for projects that were set up before Bazel modules existed—is it for other stuff too?

¹ based on https://bazel.build/rules/lib/globals/module#module:~:text=This%20can%20be%20specified%20to%20ease%20migration%20for%20projects%20that%20have%20been%20using%20a%20repo%20name%20for%20itself%20that%20differs%20from%20its%20module%20name.

Reply:
I have that in there because etc/proto is bazel-ified but src/ isn't, and etc/proto depends on stuff in src/.  Gazelle generates dependencies like `"@com_github_pachyderm_pachyderm_v2//src/protoextensions:protoextensions_go"`, which is not correct, but works for now and the problem likely goes away when src/ gets converted.